pkg/controllers/garbagecollector/garbagecollector.go

```diff
@@ -273,6 +273,7 @@ func isJobFinished(job *v1alpha1.Job) bool {
 	return job.Status.State.Phase == v1alpha1.Completed ||
 		job.Status.State.Phase == v1alpha1.Failed ||
 		job.Status.State.Phase == v1alpha1.Terminated
+		job.Status.State.Phase == v1alpha1.Aborted
```

Comment:
I have a small question: How do we distinguish between "Aborted", "Terminated" and "Failed"? Are these three states the final states of a job?

Reply:
That's a good question, `Aborted` is triggered by user, and user can set `Aborted` action in job spec.policy and pods under the pod will be evicted and job can be resumed,  `Terminated` is also set by user but from the code comments that job under this state can not be resumed, while `Failed` is  a state that pods under a job execution failed.
I'm not sure whether whethe a job under `Aborted` state can be resumed, if can be resumed, we should re-consider whether the pr is reasonable: ）

Reply:
Yes, this is critical, which determines whether it is the final state. If not, it should not be recycled.

Reply:
I think this can be close: )